### PR TITLE
Searching routes from api

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/shared/services/searchConnection.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import apiClient from './apiClient';
+
+function convertPlaces(Places) {
```
Comment (mi-chal) [inline src/shared/services/searchConnection.js]: should be `places`

File: src/shared/services/searchConnection.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  return Places.reduce(
+    (places, { PlaceId, SkyscannerCode }) => ({
+      ...places,
+      [PlaceId]: { name: SkyscannerCode },
+    }),
+    {},
+  );
+}
+
+function convertCarriers(Carriers) {
```
Comment (mi-chal) [inline src/shared/services/searchConnection.js]: Same here

File: src/shared/services/searchConnection.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  return Carriers.reduce(
+    (carriers, { CarrierId, Name }) => ({
+      ...carriers,
+      [CarrierId]: { name: Name },
+    }),
+    {},
+  );
+}
+
+function convertQuotes(Quotes) {
```
Comment (mi-chal) [inline src/shared/services/searchConnection.js]: And here :)

**Review by mi-chal — APPROVED** [review]

File: src/shared/services/searchConnection.js (hunk 1 above)
Comment (dwantula) [reply src/shared/services/searchConnection.js]: Fixed

File: src/shared/services/searchConnection.js (hunk 2 above)
Comment (dwantula) [reply src/shared/services/searchConnection.js]: Fixed

File: src/shared/services/searchConnection.js (hunk 3 above)
Comment (dwantula) [reply src/shared/services/searchConnection.js]: Fixed

**Pull request merged**